cloud/deploys/dev_azure/.terraform.tfstate.lock.info

```diff
@@ -0,0 +1 @@
+{"ID":"2e6cd317-23a1-73ec-6b7e-074d38416874","Operation":"OperationTypeApply","Info":"","Who":"zaralouis@zaralouis-macbookair.roam.corp.google.com","Version":"1.1.3","Created":"2022-02-09T23:09:17.622504Z","Path":"terraform.tfstate"}
```

Comment:
question(blocking): should this file be gitignored?

Reply:
I added that to the gitignore and removed the file from this PR